src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,34 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.9.0
+12 June, 2023
+
+### Deprecation Notice
+
+- Support for Kubernetes [1.23](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md) has been deprecated and will be removed in the next release.
+
+### Features
+
+- Kubernetes supported versions: [1.26](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.26.md), [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md) and [1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+- Upgraded the Okteto CLI version to [2.16.2](https://github.com/okteto/okteto/releases/tag/2.16.2)
+- Telemetry job is now launched as soon as the helm installation is completed.
+- Admins can now check when it was the last interaction a user had with Okteto in the Admin dashboard, under the Users section.
+- Support to access Okteto on `127.0.0.1`
+- Upgrade ingress nginx to 4.5 
+
+### Improvements
+
+- Frontend now loads faster with improved nginx compression
```

Comment:
```suggestion
- Faster load times on frontend app by improved nginx compression
```

BTW, could we consider removing the other _"now"_ usages in the PR? The "now" is implied through the change and new release, so I don't think such word usages really add any more value to the sentence? 

[More examples from past release notes](https://github.com/search?q=repo%3Aokteto%2Fdocs+now+path%3A%2F%5Esrc%5C%2Fcontent%5C%2Fself-hosted%5C%2Finstall%5C%2Freleases.mdx%2F&type=code) that similarly could be tidied up - Not in this PR, but calling it out for comparison and for future consideration.

cc @omnomagonz I'm wondering if we could consider documenting this and provide some guidelines/best-practices on how to articulate release notes more consistently in the future?